src/NUnitEngine/nunit.engine/Internal/ResultHelper.cs

```diff
@@ -46,6 +45,10 @@ public static class ResultHelper
         /// Aggregate the XmlNodes under a TestEngineResult into a single XmlNode.
         /// </summary>
         /// <param name="result">A new TestEngineResult with xml nodes for each assembly or project</param>
+        /// <param name="elementName"></param>
```

Comment:
I am all for adding missing parameters to the XML docs, but you should try to spend a bit of time reading through the code to understand what they are and add as much of a comment with them as possible. Adding the params without comments doesn't add much to the code base other than indicate that they need to be filled in. If nobody is back in this code for a long time, they might not get updated.